python/django/security/hashids-with-django-secret.yaml

```diff
@@ -0,0 +1,26 @@
+rules:
+- id: hashids-with-django-secret
+  languages:
+  - python
+  message: >-
+    The Django secret key is used as salt in HashIDs. The HashID mechanism is not secure.
+    By observing sufficient HashIDs, the salt used to construct them can be recovered.
+    This means the Django secret key can be obtained by attackers, through the HashIDs.
+  metadata:
+    category: security
```

Comment:
Looks like you are missing a subcategory

https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/#subcategory